rust/template/main.rs

```diff
@@ -35,6 +32,7 @@ use std::io::Write;
 //extern crate cpuprofiler;
 //use cpuprofiler::PROFILER;
 
+#[allow(clippy::let_and_return)]
```

Comment:
In what line does clippy about let-and-return?

Reply:
56. We need that let if the profiling code is uncommented